platform-sdk/swirlds-config-api/src/main/java/com/swirlds/config/api/ConfigurationBuilder.java

```diff
@@ -62,23 +62,15 @@ public interface ConfigurationBuilder {
      * Adds a converter (see {@link ConfigConverter}). If this method is called after the config has been created (see
      * {@link #build()}) a {@link IllegalStateException} will be thrown.
      *
+     * @param converterType the type to convert to
      * @param converter the converter that should be used for the configuration
      * @return the builder instance (useful for fluent API)
      * @throws IllegalStateException if this method is called after the config has been created
      */
     @NonNull
-    ConfigurationBuilder withConverter(@NonNull final ConfigConverter<?> converter) throws IllegalStateException;
-
-    /**
-     * Adds converters (see {@link ConfigConverter}). If this method is called after the config has been created (see
-     * {@link #build()}) a {@link IllegalStateException} will be thrown.
-     *
-     * @param converters the converters that should be used for the configuration
-     * @return the builder instance (useful for fluent API)
-     * @throws IllegalStateException if this method is called after the config has been created
-     */
-    @NonNull
-    ConfigurationBuilder withConverters(@NonNull final ConfigConverter<?>... converters) throws IllegalStateException;
```

Comment:
Why have you removed methods? adding the new one makes sense but I do not get why you remove methods.

Reply:
added them back and marked as deprecated